definitions/npm/cb-fetch_v1.11.x/flow_v0.37.x-v0.103.x/cb-fetch_v1.11.x.js

```diff
@@ -0,0 +1,123 @@
+declare module 'cb-fetch' {
+
+    declare module.exports: typeof request;
+
+    declare type BodyInit = string | URLSearchParams | FormData | Blob | ArrayBuffer | $ArrayBufferView | ReadableStream
```

Comment:
Why the override here? It's not done in the `104.x` def file, and seems to be in Flow core for a long time.

Reply:
Because it is not supported by 0.37–0.41.